### ISSUE TITLE
vscode/releases page missing binaries

### ISSUE THREAD

**Issue by citrusui** [microsoft/vscode#1230]
Just wondering... why doesn't the [releases](https://github.com/Microsoft/vscode/releases) page have vscode binaries? I only see the source code downloads. [Atom](https://github.com/atom/atom/releases) and [Brackets](https://github.com/adobe/brackets) both use GitHub to publish binaries...

Would be handy in case someone wanted to download an older release or if Microsoft's servers were down.

Inspired by [#1084](https://github.com/Microsoft/vscode/issues/1084)

**Comment by hashhar** [comment]
This is a good idea. Would make it much easier for me for to use my download utility.

**Comment by egamma** [comment]
@chrisdias fyi. 

We need to distinguish between vscode and Visual Studio Code [see here for an explanation](https://github.com/Microsoft/vscode/issues/60#issuecomment-161792005). 
Currently we do not make milestone builds available for vscode. Something we can consider.

**Comment by hoovercj** [comment]
Is this sufficient to close the issue? https://github.com/Microsoft/vscode/wiki/Previous-Releases

**Comment by citrusui** [comment]
It's just odd how there's no exe or zips on the Releases page... pretty easy to do, just add a changelog and tag, then upload the files.

**Comment by hashhar** [comment]
@hoovercj It's good enough I guess. It's also machine parseable.

**Comment by cwhits** [comment]
@hoovercj no, that's not sufficient (nor does it exist any more). I'd specifically like to be able to use the releases api to check for and download latest releases. Currently it returns nothing: https://api.github.com/repos/Microsoft/vscode/releases

**Comment by redtux** [comment]
Hi everybody, I just had a look for binary releases of VS Code ("OSS") in this git repository. While I found tarballs of the source at https://github.com/microsoft/vscode/releases there are no .deb, .rpm, .exe, or .dmg files like in other similar projects. What would be necessary for this to happen? How can we help?

JFYI: I have already read all related issues (also concerning licensing, the marketplace, etc.), but found no answer to my question. The only thing I found was this issue from December 2015.

A good example for what this could look like is GitHub's Atom (now also part of the Microsoft family). Build scripts can be found at VS Codium (that I do not belong to). CI building can be done automagically (e.g. with every new tag) using GitHub Actions (like Jenkins, GitLab CI, Travis).

- https://github.com/atom/atom/releases
- https://atom.io/releases
- https://github.com/VSCodium/vscodium
- https://github.com/actions/create-release

**Comment by HydroPlume** [comment]
While a direct yes or no answer on this topic from someone more involved in the repository would be nice, I can see why it hasn't been a priority.  Both the [Microsoft branded binaries](https://code.visualstudio.com/Download) and the [binaries built directly from this repo](https://github.com/VSCodium/vscodium/releases) are already easily accessible, the latter of which is supplied on the VSCodium project earlier mentioned.

I definitely agree that having these binaries available directly in this repository would be more organized and convenient than relying on another repository.  As it stands, the binaries produced by the VSCodium project are almost no different than would be made by CI.  Either way, after 4 years this issue should be given a definitive answer either way.  Is this being worked on? or should it be closed?

**Comment by GitMensch** [comment]
@chrisdias @kieferrm I suggest to close this with an honest "won't happen, take the branded libraries with non-MIT extensions included or build on your own if you don't want to use one of the externally provided vscode build envrionments", to finally drop this 10 year old issue from the backlog.

**Comment by chrisdias** [comment]
I agree, this won't happen. We don't build binaries solely out of this repo. As you might imagine, our internal build processes are quite extensive, building everything in the stack from scratch in secure, auditable environments, layering our product customizations on top of the content in this repo. This is what we release as the "Visual Studio Code" product that you get support for, security fixes, etc. (all for free don't forget!).

Could we build and release here? Yes, but that creates yet another column in the matrix to support.

If you want to automate downloading of older versions, this doc shows the stable API:  https://code.visualstudio.com/docs/supporting/faq#_previous-release-versions
